modules/lib/typescript/index.d.ts

```diff
@@ -0,0 +1,33 @@
+/// <reference path='global.d.ts' />
+
+declare const app: {
+    name: string;
+    version: string;
+    config: Record<string, string | undefined>;
+};
+
+declare const log: {
+    debug: (...args: unknown[]) => void;
+    info: (...args: unknown[]) => void;
+    warning: (...args: unknown[]) => void;
+    error: (...args: unknown[]) => void;
+};
+
+declare const Java: {
```

Comment:
This is Nashorn stuff (internal use). Can we avoid having it here?

Reply:
Sure. Moved to root's `global.d.ts`, that will be used in compile-time and is not a part any package that will be published.